internal/cli/atlas/backup/compliancepolicy/policies/create/create.go

```diff
@@ -0,0 +1,152 @@
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package create
+
+import (
+	"context"
+	"errors"
+	"fmt"
+
+	"github.com/mongodb/mongodb-atlas-cli/internal/cli"
+	"github.com/mongodb/mongodb-atlas-cli/internal/config"
+	"github.com/mongodb/mongodb-atlas-cli/internal/file"
+	"github.com/mongodb/mongodb-atlas-cli/internal/flag"
+	store "github.com/mongodb/mongodb-atlas-cli/internal/store/atlas"
+	"github.com/mongodb/mongodb-atlas-cli/internal/usage"
+	"github.com/spf13/afero"
+	"github.com/spf13/cobra"
+	atlasv2 "go.mongodb.org/atlas-sdk/v20230201004/admin"
+)
+
+type Opts struct {
+	cli.GlobalOpts
+	cli.WatchOpts
+	store  store.CompliancePolicyPoliciesCreator
+	fs     afero.Fs
+	path   string
+	policy *atlasv2.DataProtectionSettings
+}
+
+const (
+	active = "ACTIVE"
+)
+
+var updateTemplate = `Your backup compliance policy is being updated with the following policies:
+ID	FREQUENCY INTERVAL	FREQUENCY TYPE	RETENTION
+{{- range .ScheduledPolicyItems}}
+{{.Id}}	{{if eq .FrequencyType "hourly"}}{{.FrequencyInterval}}{{else}}-{{end}}	{{.FrequencyType}}	{{.RetentionValue}} {{.RetentionUnit}}
+{{- end}}
+{{if .OnDemandPolicyItem}}{{.OnDemandPolicyItem.Id}}	-	{{.OnDemandPolicyItem.FrequencyType}}	{{.OnDemandPolicyItem.RetentionValue}} {{.OnDemandPolicyItem.RetentionUnit}}{{end}}
+`
+var updateWatchTemplate = `Your backup compliance policy has been updated with the following policies:
+ID	FREQUENCY INTERVAL	FREQUENCY TYPE	RETENTION
+{{- range .ScheduledPolicyItems}}
+{{.Id}}	{{if eq .FrequencyType "hourly"}}{{.FrequencyInterval}}{{else}}-{{end}}	{{.FrequencyType}}	{{.RetentionValue}} {{.RetentionUnit}}
+{{- end}}
+{{if .OnDemandPolicyItem}}{{.OnDemandPolicyItem.Id}}	-	{{.OnDemandPolicyItem.FrequencyType}}	{{.OnDemandPolicyItem.RetentionValue}} {{.OnDemandPolicyItem.RetentionUnit}}{{end}}
+`
+var example = `How to run atlas backups compliancepolicy policies create with --file.
+As an example, the file should be in the following format:
+
+{
+	"frequencyInterval": 1,
+	"frequencyType": "daily",
+	"id": "stringstringstringstring",
+	"retentionUnit": "days",
+	"retentionValue": 0
+}
+
+To get the ID of a policy item, run "atlas backups compliancepolicy policies describe".
+
+Finally, run the command as such: "atlas backups compliancepolicy policies update --file /path/to/file"
+`
+var errorCode500Template = `received an internal error on the server side, but we would encourage you to double check your inputs.
+For this command, invalid inputs are known to cause internal errors in some situations`
```

Comment:
why is the 500? json files is not formatted correctly? missing fields? incompatible fields?

Reply:
invalid fields e.g. retentionUnit: "dias"
Just did some testing and also realised that
> 		if atlasv2.IsErrorCode(err, "500") {
> 			return fmt.Errorf("%v: %w", errorCode500Template, err)
> 		}

does not pick up `Error: couldn't create policy item: https://cloud-dev.mongodb.com/api/atlas/v2/groups/64b023d236931762b0c8d36b/backupCompliancePolicy PUT: HTTP 500 Internal Server Error (Error code: "UNEXPECTED_ERROR") Detail: Unexpected error. Reason: Internal Server Error. Params: []`

Reply:
>does not pick up 

Oh yeah I assume that looked at the httpcode but I think it looks at the atlas code so it needs to be 
`atlasv2.IsErrorCode(err, "UNEXPECTED_ERROR")`